flows/aws/interactive-workflow/deploy-interactive-workflow.py

```diff
@@ -0,0 +1,10 @@
+from prefect import flow
```

Comment:
Should this file just be named `deploy.py`? That follows the convention set in the datalake example.

Reply:
Good idea! I'll change the naming convention so its more consistent